src/ZPublisher/tests/test_WSGIPublisher.py

```diff
@@ -469,6 +469,48 @@ def _publish(request, module_info):
         finally:
             HTTPRequest.retry_max_count = original_retry_max_count
 
+    def test_retry_ConflictError(self):
+        # Make sure requests are retried as often as configured
+        from zope.interface import directlyProvides
+        from zope.publisher.browser import IDefaultBrowserLayer
+        registerExceptionView(ConflictError)
```

Comment:
I think the change you made ist not specific for `ConflictError` it should work for every exception class (besides `Unauthorized`), so I'd prefer using another exception class here.

Reply:
I have changed the registration to just `Exception`. I have also renamed the test method with a more descriptive name and added a better comment at the top.